ui/src/hooks/stores.ts

```diff
@@ -344,17 +344,6 @@ export const useSettingsStore = create(
   ),
 );
 
-export interface DeviceSettingsState {
```

Comment:
This was incompletely removed by some of @adam refactor to [eliminate the mouse sensitivity](https://github.com/jetkvm/kvm/pull/477) as I [just noted](https://github.com/jetkvm/kvm/pull/477#issuecomment-2897072810) on that PR